examples/documentation_examples/examples.go

```diff
@@ -2097,3 +2097,262 @@ func ChangeStreamExamples(t *testing.T, db *mongo.Database) {
 
 	atomic.StoreInt32(&stop, 1)
 }
+
+// AggregationExamples contains examples of aggregation operations.
+func AggregationExamples(t *testing.T, db *mongo.Database) {
+	ctx := context.Background()
+
+	coll := db.Collection("sales")
+	coll1 := db.Collection("air_alliances")
+
+	err := coll.Drop(context.Background())
```

Comment:
Once data is inserted through examples_tests.go, I assume these will go away.

Reply:
Gone!